src/java.base/share/classes/java/util/stream/DoublePipeline.java

```diff
@@ -263,43 +264,46 @@ public final DoubleStream flatMap(DoubleFunction<? extends DoubleStream> mapper)
                 StreamOpFlag.NOT_SORTED | StreamOpFlag.NOT_DISTINCT | StreamOpFlag.NOT_SIZED) {
             @Override
             Sink<Double> opWrapSink(int flags, Sink<Double> sink) {
-                return new Sink.ChainedDouble<>(sink) {
-                    // true if cancellationRequested() has been called
-                    boolean cancellationRequestedCalled;
-
-                    // cache the consumer to avoid creation on every accepted element
-                    DoubleConsumer downstreamAsDouble = downstream::accept;
-
-                    @Override
-                    public void begin(long size) {
-                        downstream.begin(-1);
-                    }
+                final DoubleConsumer fastPath =
```

Comment:
this `final` is unecessary, inconsistent with the style of all other fiels of this package.

Reply:
I think it is good practice to `final` anything which will be closed over. (Effectively Final is not as clear as Explicitly Final)